README.md

```diff
@@ -64,29 +121,17 @@ The example DAG can be exercised in one of two ways:
 1. Initialize a project with the [Astro CLI](https://www.astronomer.io/docs/cloud/stable/develop/cli-quickstart)
 2. Copy the example DAG into the `dags/` folder of your astro project
 3. Copy the directories in the `include` folder of this repository into the `include` directory of your Astro project
-4. Copy your GCP `credentials.json` file into the base directory of your Astro project
-5. Add the following to your `Dockerfile` to install the `airflow-provider-great-expectations` package, enable xcom pickling, and add the required Airflow variables and connection to run the example DAG:
+4. Add the following to your `Dockerfile` to install the `airflow-provider-great-expectations` package, and add the required Airflow variables and connection to run the example DAG:
 
    ```
    RUN pip install --user airflow_provider_great_expectations
-   ENV AIRFLOW__CORE__ENABLE_XCOM_PICKLING=True
-   ENV GOOGLE_APPLICATION_CREDENTIALS=/usr/local/airflow/credentials.json
-   ENV AIRFLOW_VAR_MY_PROJECT=<YOUR_GCP_PROJECT_ID>
-   ENV AIRFLOW_VAR_MY_BUCKET=<YOUR_GCS_BUCKET>
-   ENV AIRFLOW_VAR_MY_DATASET=<YOUR_BQ_DATASET>
-   ENV AIRFLOW_VAR_MY_TABLE=<YOUR_BQ_TABLE>
-   ENV AIRFLOW_CONN_MY_BIGQUERY_CONN_ID='google-cloud-platform://?extra__google_cloud_platform__scope=https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fbigquery&extra__google_cloud_platform__project=bombora-dev&extra__google_cloud_platform__key_path=%2Fusr%2Flocal%2Fairflow%2Fairflow-gcp.bombora-dev.iam.gserviceaccount.com.json'
    ```
 
 6. Run `astro dev start` to view the DAG on a local Airflow instance (you will need Docker running)
 
 **With a vanilla Airflow installation**:
 1. Add the example DAG to your `dags/` folder
-2. Make the `great_expectations` and `data` directories in `include/` available in your environment.
```

Comment:
this line needs to be added back, since we need the `include/data` directory

Reply:
[ba4d4c7](https://github.com/astronomer/airflow-provider-great-expectations/pull/161/commits/ba4d4c73480aae0172a026fa5e24ec0b2ccd76a0)